hw2/03-count.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>03 - Count</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="03-count.js"></script>
+  </head>
+  <body class="bg-info">
+    <main class="border rounded-3 bg-light w-50 mx-auto p-4 mt-5">
+      <h1 class="w-50 mx-auto">Exercise 03 - Count</h1>
+      <hr />
+      <div>
+        <label>Highlight all occurences of a word:</label>
```

Comment:
Need to add a 'for' to this label so it is connected to the input. This will cause an accessibility issue.

Reply:
Fixed.